base/tks/src/main/java/org/dogtagpki/server/tks/rest/TPSConnectorService.java

```diff
@@ -316,20 +318,23 @@ public Response createSharedSecret(String id) {
                 throw new BadRequestException("Shared secret already exists");
             }
 
-            CryptoUtil.createSharedSecret(nickname);
+            CryptoUtil.createSharedSecret(nickname,KeyGenAlgorithm.AES,AES_SESS_KEYSIZE);
 
             TPSConnectorConfig tpsConfig = cs.getTPSConnectorConfig(id);
             tpsConfig.setNickname(nickname);
             cs.commit(true);
 
-            //Create des3 session sym key to wrap the shared secret.
-            SymmetricKey tempKey = CryptoUtil.createDes3SessionKeyOnInternal();
```

Comment:
so, are we completely replacing Des3 with AES here (and a few other places)?  At some other places, you seem to be checking the key type before deciding which algorithm to use, but when it comes to creating session keys, you got rid of des3 and replaced it with AES without giving people any options.  I'm not say we should or should not keep des3.  I just think that we probably want to be consistent.